code/src/App.js

```diff
@@ -1,12 +1,18 @@
-import React from 'react'
-import data from './data.json'
+import React from "react";
+import data from "./data.json";
+import { Album } from "./Album.js";
```

Comment:
You don't need the brackets here because you are only importing one thing
```suggestion
import Album from "./Album.js";
```